test/functorch/test_aotdispatch.py

```diff
@@ -1173,6 +1173,7 @@ def assert_compiler(gm: torch.fx.GraphModule, _):
     xfail('nn.functional.rrelu', ''),  # aten.rrelu_with_noise.default - couldn't find symbolic meta function...
     xfail('nn.functional.smooth_l1_loss', ''),  # could not find kernel
     xfail('nn.functional.unfold', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
+    xfail('unfold', ''),  # aten.squeeze_copy.dim - couldn't find symbolic meta function/decomposition
```

Comment:
these xfails I would prefer to understand why they weren't previously failing. It is especially puzzling because most changes were to dynamo, yet somehow the AOTAutograd test is failing

Reply:
Yes I don't understand it either.

Reply:
The `squeeze_copy` is pretty sus - `{view}_copy` ops (except for narrow_copy) are only really meant to show up when you use the `remove='mutations_and_views'` variant of functionalization, which is not what AOTAutograd uses. When my build finishes I'm going to take a quick look.

Reply:
oh it's a dumb reason, we use `unfold_copy` in one of our decomps. Not sure why it wasn't getting hit previously: https://github.com/pytorch/pytorch/blob/c2ef5c4f7ee894e12e44af6d6aa2c4972cf71025/torch/_decomp/decompositions.py#L925